webapp/src/components/Help.tsx

```diff
@@ -14,58 +16,84 @@ import { Stack, Container, useTheme } from '@mui/material'
 const Help = () => {
   const theme = useTheme()
   return (
-    <Container maxWidth={false} id="help" sx={{ textAlign: 'left', backgroundColor: theme.palette.background.default }}>
+    <Container
+      maxWidth={false}
+      id="help"
+      sx={{ textAlign: 'left', backgroundColor: theme.palette.background.default }}
+    >
       <Stack spacing={2}>
         <h2>Welcome to the {EnvironmentVars.DOT_NAME} CV Manager Website</h2>
```

Comment:
While we are updating this page - I would like to break the help page into sections, describing the different features. All of the components here describe the MAP page, which is fine - but that should be noted as such, with a section title